ui/src/members/components/SelectUsers.tsx

```diff
@@ -15,22 +16,39 @@ interface Props {
 
 export default class SelectUsers extends PureComponent<Props> {
   public render() {
-    const {users} = this.props
+    return <>{this.dropdownUsers}</>
+  }
+  private get dropdownUsers() {
+    const {users, selectedUserIDs, onSelect} = this.props
 
-    return (
-      <>
+    if (!users || !users.length) {
```

Comment:
I think you could make getters for the `emptyText` and `status` props that use this logic, rather than having to ensure that changes to `MultiSelectDropdown` don't have to be made in two places